python/paddle/fluid/transpiler/memory_optimization_transpiler.py

```diff
@@ -497,7 +497,7 @@ def memory_optimize(input_program,
                     skip_opt_set=None,
                     print_log=False,
                     level=0,
-                    skip_grads=False):
+                    skip_grads=True):
```

Comment:
There is no comments for this arg. And why change to True?

Reply:
`memory_optimize` reduce the memory to use by renaming the var's name, in some case, we don't want the name of parameters' gradient to be renamed, otherwise, we could not find the gradient of the parameter.